src/service-worker/ServiceWorker.ts

```diff
@@ -202,11 +196,11 @@ export class ServiceWorker {
                 // Never nest the following line in a callback from the point of entering from retrieveNotifications
                 notificationEventPromiseFns.push((notif => {
                   return ServiceWorker.displayNotification(notif)
-                      .then(() => ServiceWorker.updateBackupNotification(notif).catch(e => Log.error(e)))
                       .then(() => {
                         return ServiceWorker.workerMessenger.broadcast(WorkerMessengerCommand.NotificationDisplayed, notif).catch(e => Log.error(e))
```

Comment:
Linter complains about line length

Reply:
This line wasn't changed in this PR, thinking we should leave as is for now as this whole method needs a refactor